.github/workflows/darwin.yaml

```diff
@@ -162,6 +162,11 @@ jobs:
                   # but to instrument the code in the underlying libCHIP we need to pass CHIP_IS_TSAN=YES
                   xcodebuild test -target "Matter" -scheme "Matter Framework Tests" -sdk macosx -enableThreadSanitizer YES OTHER_CFLAGS='${inherited} -Werror -Wconversion -Wno-incomplete-umbrella -Wno-unguarded-availability-new' CHIP_IS_TSAN=YES > >(tee /tmp/darwin/framework-tests/darwin-tests-tsan.log) 2> >(tee /tmp/darwin/framework-tests/darwin-tests-tsan-err.log >&2)
               working-directory: src/darwin/Framework
+            - name: Build Matter TV Casting Bridge
+              timeout-minutes: 45
```

Comment:
I worry about this hitting the overall job timeout _and_ giving us extremely long CI latency, if this thing really has a 45-minute timeout.

Also, this does not have anything to do with Matter.framework, which is what all the other tests here are about.

But most importantly, it's not clear how this fixes the "pushing past limits" aspect of this PR.

Reply:
This will lower latency, we should put it here. Agree it should be a better PR. I'll lower the timeout required, but the previous location had it spinning for a wasted 30+ minutes in CI, that would be amortized here.

Reply:
Alright, if this job is actually pretty quick, putting it here is probably fine.